vm/src/tests/cairo_run_test.rs

```diff
@@ -1240,3 +1240,54 @@ fn cairo_run_apply_poly_proof() {
         include_bytes!("../../../cairo_programs/mod_builtin_feature/proof/apply_poly.json");
     run_program(program_data, true, None, None, None);
 }
+
+#[test]
+#[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+fn cairo_run_secp_cairo0_reduce_value() {
+    let program_data = include_bytes!("../../../cairo_programs/secp_cairo0_reduce_value.json");
+    run_program_simple(program_data.as_slice());
+}
```

Comment:
The CI seems to be failing because these tests are being run even without the required feature flag. Could you add `#[cfg(feature = "cairo-0-secp-hints")]` to these tests?

Reply:
This has been fixed.